.infra/crons.ts

```diff
@@ -43,5 +43,9 @@ export const crons: Cron[] = [
   {
     name: 'refresh-user-subscriptions',
     schedule: '0 */3 * * *',
+  },
+  {
+    name: 'personalized-digest',
+    schedule: '0 0 * * 0'
```

Comment:
Currently running once a week on Sunday to generate Monday digest. Additional logic shall be added once we integrate user preference for which day they want the digest to send.